homeassistant/components/starline/lock.py

```diff
@@ -0,0 +1,55 @@
+"""Support for StarLine lock."""
+from homeassistant.components.lock import LockDevice
+from .account import StarlineAccount, StarlineDevice
+from .const import DOMAIN
+from .entity import StarlineEntity
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the StarLine lock."""
+
+    account: StarlineAccount = hass.data[DOMAIN][entry.entry_id]
+    entities = []
+    for device in account.api.devices.values():
+        if device.support_state:
+            entities.append(StarlineLock(account, device))
+    async_add_entities(entities)
+    return True
+
+
+class StarlineLock(StarlineEntity, LockDevice):
+    """Representation of a StarLine lock."""
+
+    def __init__(self, account: StarlineAccount, device: StarlineDevice):
+        """Initialize the lock."""
+        super().__init__(account, device, "lock", "Security")
+
+    @property
+    def available(self):
+        """Return True if entity is available."""
+        return super().available and self._device.online
+
+    @property
+    def device_state_attributes(self):
+        """Return the state attributes of the lock."""
+        return self._device.alarm_state
```

Comment:
What does this return? Please set the dict explicitly here so we can see what is returned and the formatting.

Reply:
This `Dict[str, bool]` can contains 12 optional values. Count of values depends on security system model, car and other conditions. Is it really required to add 12 conditions here to copy values from dict to dict?

Reply:
The attribute keys must be formatted lowercase snake_case. Is that the case?

Reply:
Yes. List of possible keys ([doc](https://developer.starline.ru/#api-Device-DeviceState)):
`add_h`, `add_l`, `door`, `hbrake`, `hijack`, `hood`, `ign`, `pbrake`, `shock_h`, `shock_l`, `tilt`, `trunk`

Reply:
Ok. Please at least add a code comment explaining what the keys are.